image_uploader_widget/widgets.py

```diff
@@ -3,10 +3,13 @@
 
 class ImageUploaderWidget(widgets.ClearableFileInput):
     template_name = 'admin/widgets/image_uploader_widget.html'
+    add_image_text = ""
     drop_text = ""
     empty_text = ""
     empty_icon = ""
     drop_icon = ""
+    add_icon = ""
+    accept = "image/*"
```

Comment:
this should be changed here?

Is lost in my personal clone?

Reply:
Oh, this appears is from my first try of making the widget to work. Will be removed here.